code/game/objects/items/weapons/twohanded.dm

```diff
@@ -381,12 +381,17 @@
 	sharpness = IS_SHARP
 	actions_types = list(/datum/action/item_action/startchainsaw)
 	var/on = 0
+	var/on_text = "it begins to whirr."
+	var/off_text = "the chain stops moving."
+	var/force_off = 13
```

Comment:
yes, this is true, but there's no point in defining two extremely similar variables, when the only difference is in the nomenclature of "on" vs "wielded" (which are already close anyway)